platform/ios/src/MGLMapView.mm

```diff
@@ -1948,8 +1949,9 @@ - (void)showAttribution
     
     UIViewController *viewController = self.window.rootViewController;
     if ([viewController isKindOfClass:[UINavigationController class]]) {
-        viewController = [(UINavigationController *)viewController viewControllers].firstObject;
+        viewController = ((UINavigationController *)viewController).topViewController;
     }
+    viewController = [UIViewController getTopViewControllerFrom:viewController];
```

Comment:
I think it's always if it's indeed a navigation controller, since a navigation controller with out any view controllers doesn't make any sense. Or am I misunderstanding you @frederoni?